packit_service/worker/handlers/github_handlers.py

```diff
@@ -521,13 +520,13 @@ def pre_check(self) -> bool:
         if not valid:
             return False
 
-        configured_branch = (
-            self.koji_build_helper.job_build.metadata.dist_git_branch or "master"
+        configured_branches = (
+            self.koji_build_helper.job_build.metadata.branch or "master"
         )
-        if configured_branch != self.event.git_ref:
+        if self.event.git_ref != configured_branches:
```

Comment:
This looks like you can configure multiple branches, but it can be only one `str` for now. (I don't get the reason for renaming to `configured_branches`.)

Reply:
Do we want to support also a list of branch-names? That's not a bad idea.

Reply:
Oh, I see. Thats a long story, I'll rather create a follow-up PR :-)

Reply:
Will be resolved in #568